src/i18n/translations.ts

```diff
@@ -5,6 +5,134 @@ export const languages = {
 
 export const defaultLang = 'de';
 
+export const serviceTranslations = {
+    'services.title': {
+        de: 'Unsere Dienstleistungen',
+        en: 'Our services',
+    },
+    'services.intro': {
+        de: 'Wir bieten Ihnen maßgeschneiderte Lösungen für Ihre Anforderungen. Profitieren Sie von der Multidisziplinarität unseres Teams. Wir bringen Kompetenzen aus Geographie und Informatik ein, um Probleme fachlich korrekt und effizient zu lösen. ',
+        en: 'We offer you customized solutions for your requirements. Benefit from the multidisciplinarity of our team. We bring together expertise in geography and computer science to solve problems correctly and efficiently.',
+    },
+    'services.advantages': {
+        de: 'Ihre Vorteile',
+        en: 'Your advantages',
+    },
+    'services.esg.heading': {
```

Comment:
should this really be #1 of our services? why not start with "We support you with your projects"

Reply:
nobody cared, yet

did not work out I guess

Reply:
maybe we should change our main slogan on the start page then as well.

Your #1 source for Biodiversity in ESG Reporting

order sowas

Reply:
060f4560ec2adf11cee2684c991dff852ff4d34a